src/instructlab/model/download.py

```diff
@@ -135,11 +135,21 @@ class OCIDownloader(Downloader):
     We are leveraging OCI v1.1 for this functionality
     """
 
-    def __init__(self, repository: str, release: str, download_dest: str, ctx) -> None:
-        super().__init__(
-            repository=repository, release=release, download_dest=download_dest
-        )
-        self.ctx = ctx
+    @staticmethod
+    def _load_json(file_path: Path):
```

Comment:
@booxter couldn't this generic function to read json files be used elsewhere in the project as well? That was the main reason I'd put it in the `utils.py` file

Reply:
That may be true (though it's not used anywhere else right now). Then the name of the function should not be private (remove `_`)?

Reply:
That would be my preference anyway
I can think of a couple other places we could use this function

Reply:
I will return the load_json one back; and will leave extract_sha in place since it's very specific to OCI.